challenge_4/go/erocs/invert_tree.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+)
+
+type TreeNode struct {
+	Left  *TreeNode
+	Right *TreeNode
+	Value int
+}
+
+func (t *TreeNode) String() string {
+	l := ""
```

Comment:
(Also yes I do appreciate the irony of 1-letter variable names when talking about descriptiveness. Once again, laziness in my part to crank out a simple problem, lol. I'm glad code standards will be implemented in the repo.)